db/migrate/20140227151213_add_portal_run_count_to_lightweight_activity.rb

```diff
@@ -5,7 +5,7 @@ class Tun < ApplicationRecord
 
   class Act < ApplicationRecord
     self.table_name = 'lightweight_activities'
-    has_many :runs, foreign_key: 'activity_id', class_name: Tun
+    has_many :runs, foreign_key: 'activity_id', class_name: "Tun"
```

Comment:
Shouldn't it be "Run" and not "Tun"?

Reply:
Nevermind.  I see it is defined locally and I get the joke too "Act Tun" (Achtung).